core/common/src/main/java/alluxio/grpc/GrpcService.java

```diff
@@ -52,6 +57,29 @@ public GrpcService disableAuthentication() {
     return this;
   }
 
+  /**
+   * Add a new closeable resource to this service's closer.
```

Comment:
who is responsible for closing this? the caller or callee?

Reply:
`GrpcServer` owns `GrpcService` and closing its closer.

Reply:
can you add a comment somewhere to explicitly mention which class owns this closer and will close it?